velox/docs/functions/spark/comparison.rst

```diff
@@ -75,5 +75,32 @@ Comparison Functions
     Returns true if x is not equal to y. Supports all scalar types. The types
     of x and y must be the same. Corresponds to Spark's operator ``!=``.
 
+.. spark:function:: decimal_lt(x, y) -> boolean
```

Comment:
Is it necessary to use different function names? Can we not use lt for both decimal and non-decimal inputs? You can register function with same name twice - once as a simple function and another time as a vector function - as long as signatures are different.

Reply:
They are registered as vector functions now, so different names are needed.
[RegisterCompare.cpp#L24-L35](https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/RegisterCompare.cpp#L24-L35)